src/app/delegation_backend/src/itn_uptime_analyzer/identity.go

```diff
@@ -0,0 +1,129 @@
+package itn_uptime_analyzer
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"strconv"
+	"strings"
+	"time"
+
+	dg "block_producers_uptime/delegation_backend"
+
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	logging "github.com/ipfs/go-log/v2"
+	sheets "google.golang.org/api/sheets/v4"
+)
+
+type Identity map[string]string
```

Comment:
Is there a particular reason to have this type defined via `map` and not as a regular `struct` with fields?

Reply:
Not really, other than the lack of experience with structs.